FindOnPage.md

```diff
@@ -0,0 +1,664 @@
+        if (webView.CoreWebView2 == null)
+        {
+            throw new InvalidOperationException("WebView2 is not initialized.");
+        }
+
+        // Initialize the find configuration with specified settings.
+        var findConfiguration = new CoreWebView2FindConfiguration
+        {
+            FindTerm = findTerm,
+            IsCaseSensitive = false,
+            ShouldMatchWord = false,
+            FindDirection = CoreWebView2FindDirection.Forward
+        };
+
+        // Specify that a custom UI will be used for the find operation.
+        webView.CoreWebView2.Find.SuppressDefaultDialog = true;
+        webView.CoreWebView2.Find.ShouldHighlightAllMatches = true;
+
+        // Start the find operation with the specified configuration.
+        await webView.CoreWebView2.Find.StartFindAsync(findConfiguration);
+        // It's expected that the custom UI for navigating between matches (next, previous)
+        // and stopping the find operation will be managed by the developer's custom code.
+    }
+    catch (Exception ex)
+    {
+        // Handle any errors that may occur during the find operation.
+        Console.WriteLine($"An error occurred: {ex.Message}");
+    }
+}
+//! [ConfigureAndExecuteFindWithCustomUI]
+```
+
+### Retrieve the Index of the Active Match
+    
+#### Description
+Developers can retrieve the index of the currently active match 
+within a WebView2 control using the `GetActiveMatchIndex` method.
+    
+    
+```cpp
+    //! [GetActiveMatchIndex]
+    bool AppWindow::GetActiveMatchIndex()
+    {
+        auto webView2_17 = m_webView.try_query<ICoreWebView2_17>();
+        CHECK_FEATURE_RETURN(webView2_17);
+        wil::com_ptr<ICoreWebView2Find> webView2find;
+        CHECK_FAILURE(webView2_17->get_Find(&webView2Find));
+        LONG activeMatchIndex;
+        CHECK_FAILURE(webView2Find->get_ActiveMatchIndex(&activeMatchIndex));
+    
+        // Update UI or handle activeMatchIndex as you wish
+        // For example, you could show a message box
+        std::wstring activeMatchIndexStr =
+            L"Active Match Index: " + std::to_wstring(activeMatchIndex);
+        MessageBox(m_mainWindow, activeMatchIndexStr.c_str(), L"Find Operation", MB_OK);
+    
+        return true;
+    }
+    //! [GetActiveMatchIndex]
+
+    // Register ActiveMatchIndexChanged event handler
+    m_webView->add_ActiveMatchIndexChanged(
+        Callback<ICoreWebView2FindActiveMatchIndexChangedEventHandler>(
+            [this](LONG activeMatchIndex) -> HRESULT
+            {
+                // Update custom UI 
+                wprintf(L"Active Match Index Changed: %ld\n", activeMatchIndex);
+                return S_OK;
+            }).Get(),
+        &m_activeMatchIndexChangedToken);
+```
+#### .NET C#
+```csharp
+//! [GetActiveMatchIndex]
+public async Task<int> GetActiveMatchIndexAsync()
+{
+    var webViewFind = webView.CoreWebView2.Find; // Assuming webView is your WebView2 control
```

Comment:
Sample nit: make it a class member and declare it

Reply:
Please fix (make it _webView)